esmvaltool/cmorizers/obs/cmor_config/WFDE5.yml

```diff
@@ -0,0 +1,47 @@
+---
+# Filename (will be extended by variable name and reference)
+filename: '{raw_name}_WFDE5_{reference}_*_v1.0.nc'
+
+# Common global attributes for Cmorizer output
+attributes:
+  dataset_id: WFDE5
+  version: 'v1.0'
+  tier: 2
+  modeling_realm: ground
+  project_id: OBS
+  source: 'https://doi.org/10.24381/cds.20d54e34'
+  reference: 'wfde5'
+  comment: ''
+
+# Variables to cmorize
+variables:
+  tas_CRU:
+    short_name: tas
+    raw_name: Tair
+    reference: CRU
+    mip: hour
+    raw_units: K
+    calendar: gregorian
+    save_hourly: False
+    add_day: True
+    add_mon: True
+  pr_CRU-GPCC:
+    short_name: pr
```

Comment:
afaik total precipitation (pr) is the sum of rainfall and snowfall. snowfall is available in WFDE5 as well. Wouldn't it make sense to sum them up?

Reply:
Thanks for pointing this out @lukruh ! I'll change it next week.